### DISCUSSION TITLE
stream in local network

### DISCUSSION THREAD

**Discussion by maxixam** [HaishinKit/HaishinKit.swift#1113] [Q&A]
i have a sample code for HaishinKit that is connected to GCore.  I am simply trying to use the phone as a hotspot and use local address to access the stream.  Is there an easy way to do that?  Thanks.

**Comment by shogo4405** [comment]
I'm using for testing for localhost.
- Wowza Server
  - https://www.wowza.com/
- nginx + nginx-rtmp-module
  - https://github.com/arut/nginx-rtmp-module/
